docker/compose/mozdef_sampledata/files/supervisor.conf

```diff
@@ -0,0 +1,9 @@
+[supervisord]
+nodaemon=true
+autostart=true
+autorestart=true
+#real time constant stream of demo events/alerts/attackers
+[program:realTimeEvents]
+priority=10
+command=bash -c 'sleep 90 && source /opt/mozdef/envs/python/bin/activate && /opt/mozdef/envs/mozdef/examples/demo/sampleevents.sh'
```

Comment:
Do we need to use supervisor here since we're only running one command? Can this be done all via the command statement in docker-compose?

Reply:
Probably could be? Maybe there are benefits of leaving it to not require docker in order to run, though?